engines/ultima/detection_tables.h

```diff
@@ -31,6 +31,37 @@ namespace Ultima {
 #define GUI_OPTIONS_MARTIAN_DREAMS GUIO0()
 #define GUI_OPTIONS_SAVAGE_EMPIRE  GUIO0()
 
+// Hack so arguments containing commas work for nested macros
+#define MACRO_ARG(...) __VA_ARGS__
```

Comment:
Why is this needed? This is only used to wrap adFileDesc, but the comment is inconsistent with its usage